models/user_model.py

```diff
@@ -93,6 +94,10 @@ def delete_user(username):
         db.session.commit()
         return done
 
+    def update_last_login(self):
```

Comment:
update_last_login should handle potential database errors during commit. Consider returning success/failure status